my_sprites.py

```diff
@@ -1,4 +1,82 @@
 import arcade
+import math
+
+class Enemy(arcade.Sprite):
+    """
+    parent class for all enemies in the game
+    """
+
+    def __init__(self, filename: str, center_x: float, center_y: float, max_hp: int, speed: float, scale=1.0):
```

Comment:
Maybe use a _position_ as input instead of x & y? I think we are moving towards doing that in general.